src/Services/ItemSearch/Factories/Faker/CategoryTreeFaker.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace IO\Services\ItemSearch\Factories\Faker;
+
+class CategoryTreeFaker extends AbstractFaker
+{
+    public function fill($data)
+    {
+        /** @var CategoryFaker $categoryFaker */
+        $categoryFaker = pluginApp(CategoryFaker::class);
+        $categoryFaker->types = ['item', 'content', 'item', 'container', 'item', 'blog'];
+        
+        for($i = 1; $i <= 10; $i++)
+        {
```

Comment:
PSR-2 Codestyling beachten
- Leerzeichen nach for
- Leerzeichen nach condition Klammer und dann geschweifte Klammer nicht in eine extra Zeile